.changeset/change.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
now we have 2 changeset files?

Reply:
no we have just one. this one needs to deleted.